inspirehep/dojson/hep/fields/bd01x09x.py

```diff
@@ -189,7 +189,7 @@ def get_value(val):
         return {
             'a': val.get('value'),
             '9': val.get('source'),
-            '2': val.get('type') or "DOI"
+            '2': val.get('schema') or "DOI"
```

Comment:
This is weird at the moment. I would split this in 2 rules. One for dois (with schema always set to DOI) and one for the other persistent_identifiers (with schema read from `schema`).